### PR TITLE
create Repo and service

### PR DESCRIPTION
**Pull request by Sardor0305**

### CODE REVIEW

File: src/main/java/uz/pdp/eufloria/domain/Order.java [hunk 1]
```diff
@@ -1,16 +1,7 @@
...
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import jakarta.persistence.Table;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
+import jakarta.persistence.*;
+import lombok.*;
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/domain/Order.java]: remove wildcard

Comment (Sardor0305) [reply src/main/java/uz/pdp/eufloria/domain/Order.java]: fixed

**Review by ismoil0709 — DISMISSED** [review]

File: src/main/java/uz/pdp/eufloria/domain/Order.java (hunk 1 above)
Comment (byte-11) [reply src/main/java/uz/pdp/eufloria/domain/Order.java]: not fixed

File: src/main/java/uz/pdp/eufloria/dto/response/OrderDto.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package uz.pdp.eufloria.dto.response;
+
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.ManyToOne;
+import lombok.*;
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/dto/response/OrderDto.java]: remove wild card from here also

Comment (Sardor0305) [reply src/main/java/uz/pdp/eufloria/dto/response/OrderDto.java]: fixed all of the code

**Review by ismoil0709 — APPROVED** [review]

**Pull request merged**